src/components/currentData.js

```diff
@@ -0,0 +1,85 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  default: {
+    currentData: {
+      float: 'left',
+      width: '49.8%',
+      marginBottom: 0,
+      backgroundColor: '#ef4856',
+      height: 240,
+    },
+    container: {
+      float: 'top',
```

Comment:
`top` is not a valid value for `float` https://developer.mozilla.org/en-US/docs/Web/CSS/float#Values